graphql/tests/repoSettings/projectRef/queries/project_ref.graphql

```diff
@@ -15,6 +15,7 @@
       patchingDisabled
       prTestingEnabled
       stepbackDisabled
+      stepbackBisection
```

Comment:
The way these tests work is they make a query and then compare the results it returns to the expected values. The expected value is in a results.json file under the name of this file (project_ref.graphl), so I usually look for it by grepping for the file name. This one is here: https://github.com/evergreen-ci/evergreen/blob/main/graphql/tests/projectSettings/projectRef/results.json#L4

Reply:
If you need to edit the data, you can do it in the data.json file in the same location as the results.json file, which in this case is [here](https://github.com/evergreen-ci/evergreen/blob/main/graphql/tests/projectSettings/projectRef/data.json).

Reply:
I think I got it, I added it to the test data and then I added it to the query too